src/app/parcels/add/formSections/TimeofDayCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { CardProps, onChangeTime, errorText } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+import StyledTimePicker from "@/components/DataInput/TimePicker";
+import { ErrorText } from "@/components/Form/formStyling";
+
+const TimeOfDayCard: React.FC<CardProps> = ({ fieldSetter, formErrors, errorSetter }) => {
+    const [timePickerValue, setTimePickerValue] = React.useState<Date | null>(null);
+    return (
+        <GenericFormCard title="Packing Time" required={true}>
```

Comment:
Add back text="What time is the parcel due to be packed?" to be consistent?

Reply:
Changed